Sources/BrowserServicesKit/Autofill/AutofillUserScript+Email.swift

```diff
@@ -96,4 +97,20 @@ extension AutofillUserScript {
         }
     }
 
+    func emailGetUserData(_ message: AutofillMessage, _ replyHandler: @escaping MessageReplyHandler) {
+        emailDelegate?.autofillUserScriptDidRequestUserData(self) { username, alias, token, _ in
```

Comment:
Is there anything to stop arbitrary web pages calling this function?

Reply:
Yes, these handlers are injected in a private context that is not available to web pages. For older devices that don't support that private context, we devised a different strategy that includes randomised method names and encryption so web pages can't access it there either.